indi-asi/asi_base.cpp

```diff
@@ -464,8 +464,11 @@ bool ASIBase::updateProperties()
         defineProperty(BlinkNP);
         defineProperty(ADCDepthNP);
         defineProperty(SDKVersionSP);
-        defineProperty(SerialNumberTP);
-        defineProperty(NicknameTP);
+        if (!mSerialNumber.empty())
```

Comment:
Need to do a similar check in the delete part below.

Reply:
Done.